xls/examples/bitonic_sort.x

```diff
@@ -0,0 +1,235 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Can you put the project license header in the .x file? That's a requirement to merge the PR, thanks!

Reply:
Added!